foundry/foundry.py

```diff
@@ -857,6 +857,53 @@ def toTorch(self, raw=None, split=None):
             return FoundryDataset_Torch(inputs, targets)
         else:
             raise NotImplementedError
+
+    def toTensorflow(self, raw=None, split=None):
+        """Convert Foundry Dataset to a Tensorflow Sequence
+
+        Arguments:
+            raw (dict): The output of running ``f.load_data(as_hdf5=False)``
```

Comment:
I'd explain what the dictionary is without referencing another method.

Reply:
I just removed that argument all together as it isn't useful, and can cause code to break